plugin-flex-ts-template-v2/src/feature-library/queues-view-data-tiles/custom-components/AllChannelsSLATile/AllChannelsSLATile.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Icon, useFlexSelector } from '@twilio/flex-ui';
+import { PieChart } from 'react-minimal-pie-chart';
+import { Data } from 'react-minimal-pie-chart/types/commonTypes';
+import AppState from 'types/manager/AppState';
+import { WorkerQueue } from '@twilio/flex-ui/src/state/QueuesState';
+
+import { getChannelIcon } from '../../utils/helpers';
+import { TileWrapper, Title, Summary, Chart, Channel, Label, Metric, SLPct } from './AllChannelsSLATile.Components';
+import QueueDataUtil from '../../utils/QueueDataUtil';
+import { SLMetrics } from '../../types';
+
+interface ComponentProps {
+  channelList: string[];
+  colors: {
+    [key: string]: string;
+  };
+}
+
+const AllChannelsSLATile = (props: ComponentProps) => {
+  const { channelList, colors } = props;
+  const sla: SLMetrics = useFlexSelector((state: AppState) => {
+    const queues: WorkerQueue[] = Object.values(state.flex.realtimeQueues.queuesList);
+    return QueueDataUtil.getSLTodayByChannel(queues, channelList);
+  });
+  const data: Data = [];
+  channelList.forEach((ch) => {
+    const handled = sla[ch]?.handledTasks || 0;
+    if (handled) data.push({ title: ch, value: handled, color: colors[ch] });
+  });
+  return (
+    <TileWrapper className="Twilio-AggregatedDataTile">
+      <Summary>
+        <Title>SLA Today</Title>
+        {channelList.map((ch) => {
+          const handled = sla[ch]?.handledTasks || 0;
+          const slPct = sla[ch]?.serviceLevelPct;
+          return (
+            <Channel key={ch}>
+              <Icon icon={getChannelIcon(ch)} />
+              <Label bgColor={colors[ch]}>{ch}</Label>
+              {handled > 0 ? <SLPct value={slPct}> {slPct}% </SLPct> : <Metric> - </Metric>}
```

Comment:
The `-` is not visible in dark mode

Reply:
Added a neutral background color (default token from here https://paste.twilio.design/tokens/list)